src/ff7/misc.cpp

```diff
@@ -248,20 +286,24 @@ struct ff7_gamepad_status* ff7_update_gamepad_status()
 			ff7_externals.gamepad_status->button5 = gamepad.IsPressed(XINPUT_GAMEPAD_LEFT_SHOULDER); // L1
 			ff7_externals.gamepad_status->button6 = gamepad.IsPressed(XINPUT_GAMEPAD_RIGHT_SHOULDER); // R1
 			ff7_externals.gamepad_status->button7 = gamepad.leftTrigger > 0.85f; // L2
-			ff7_externals.gamepad_status->button8 = gamepad.rightTrigger > 0.85f; // R2
+
+			// Remap R2 trigger to R3 so that we can use the trigger for zoom-in when analogue controls are enabled
```

Comment:
Please do not change controls if a feature is active. 
What happens if this is enabled and i want to remap w/ R2 / L2 do in battle ? 

I suggest
  Either use L3 / R3 for zoom 
                 or 
  Hold R2 or L2 and use the right stick's up / down  for zoom